Api/Data/NodeInterface.php

```diff
@@ -262,4 +263,15 @@ public function getAdditionalData();
      * @return $this
      */
     public function setAdditionalData($data);
+
+    /**
+     * @return mixed
```

Comment:
Wouldn't this return an int type?

Reply:
Thanks for spotting this, changes this with https://github.com/SnowdogApps/magento2-menu/pull/245/commits/1f9444c87ab64da5cfdf3232c9cf092e3df660ba